pkg/apis/kops/componentconfig.go

```diff
@@ -374,20 +374,21 @@ type KubeAPIServerConfig struct {
 
 	LogLevel int `json:"logLevel,omitempty" flag:"v"`
 
```

Comment:
What's with the formatting here? Not that it should matter if it's just formatting - but do we need this in the PR?